src/twisted/trial/test/erroneous.py

```diff
@@ -109,12 +124,26 @@ def test_fail(self) -> defer.Deferred[None]:
         """
         return deferLater(reactor, 0, self.fail, "I fail later")  # type: ignore[arg-type]
 
-    def test_exception(self):
+    def test_failGreaterThan64k(self) -> defer.Deferred[None]:
+        """
+        A test which fails in the callback of the returned L{defer.Deferred}
+        with a very long string.
+        """
```

Comment:
```suggestion
        """
        A test which returns a L{Failure} in the callback of the returned L{defer.Deferred}
        with a very long string.
        
        Note: there is a separate test for checking the case in which an
        exception is raised.
        
        This is triggered via L{test_addFailureGreaterThan64k}.
        """
```

Reply:
Hmm.  I'm not sure about this.  The last callback on the returned `Deferred` doesn't do `return Failure(...)` but `self.fail(...)`.  It will actually raise an exception.  Of course, this is turned into a `Failure` by `Deferred` so I'm not sure it's possible to tell the difference between these cases (except the raised exception might have more stack frames in its traceback).